compatibility/bobs/data-updates.lua

```diff
@@ -0,0 +1,66 @@
+local IC = require "prototypes.shared"
+
+if mods["boblogistics"] and mods["boblibrary"] then
+  -- IC's recipe and technology
+  bobmods.lib.recipe.replace_ingredient(IC.ENTITY_PREFIX.."1", "stack-inserter", "red-stack-inserter")
+  bobmods.lib.tech.add_prerequisite(IC.TECH_PREFIX.."1", "engine")
+  bobmods.lib.tech.add_prerequisite(IC.TECH_PREFIX.."3", "logistics-3")
+  data.raw["assembling-machine"][IC.ENTITY_PREFIX.."3"].next_upgrade = IC.ENTITY_PREFIX.."4"
+
+  -- Add containerization machines for tier 4&5 of Bob's belts
+  intermodal_containers.add_tier(
+  {
+    localised_name = "Turbo conatiner loader",
```

Comment:
In reality, this is the best place because since all compatibility use the same internal name, the localisation depends on the mod that it activates and they will just override between them. If it is in the locale, i still have to do it manually in data stage

Reply:
But you can rename everything and have tier 1-5 have all standardized names independent from other mods or belts ' name

Reply:
But presumably this approach won't allow other locale files to set the name for their own language?

Reply:
Maybe i can set it to be something like {item.kr-cont-machine} in K2 compat and {item.bob-cont-machine} in bob's, and they will just point to the locale file that can have all langs

Reply:
I see - yes that would be best. So it can be like this:
```
[item-name]
ic-containerization-machine-4-kr=Immersite container loader
ic-containerization-machine-4-bob=Turbo container loader
```
And then they can specify `localised_name = {"item-name. ic-containerization-machine-4-kr"}`